computation/Project.toml

```diff
@@ -1,4 +1,21 @@
-name = "RadiotherapyPlansBackend"
-uuid = "b4a558f9-ded2-4965-ab58-877dd48e747b"
-authors = ["trebor <rob.koziarski2k@gmail.com>"]
-version = "0.1.0"
```

Comment:
Nie jest to konieczne?

Reply:
Przeniosłem `Project.toml` z [maina](https://github.com/Project-Summer-AI-Lab-Glider/radiotherapy-plans-backend/blob/main/Project.toml), więc zakładam, że nie jest

Reply:
To byłoby konieczne do korzystania z kodu jako z pakietu, ale jeśli wystarczy nam funkcjonalność środowiska Julii to nie jest to konieczne.